request/request.js

```diff
@@ -2,6 +2,8 @@
 
 var buildQueryString = require("../querystring/build")
 
+var FILE_PROTOCOL_REGEX = new RegExp('^file://', 'i')
```

Comment:
@pygy right, I was looking at the diff only so I wasn't aware of the actual code position. 
I usually don't define such simple regular expressions as separate variables, but that might be a style preference. Also having a good syntax highlighting in your code editor would highlight the literal regular expression versus the string which will be highlighted as .. well as string :)  
My initial example from few comments above was using inline literal regexp inside the event handler, so that would have made a performance difference .. I guess.
Also not having the regex as a separate variable would definitely save up some bytes, though that wasn't my point :)